### ISSUE TITLE
change-id warning shows up as clutter in CI logs

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rust#130352]
CI logs (specifically the mingw-check PR CI job) have a lot of this:
```
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 129473` at the top of `config.toml`
NOTE: this message was printed twice to make it more likely to be seen
```
This should probably be silenced on CI to avoid cluttering the logs?
Cc @onur-ozkan

**Comment by onur-ozkan** [comment]
Interesting... We have this already for ignoring change-tracker warnings:

https://github.com/rust-lang/rust/blob/e7386b361dda8c2222bee7a1ae79faba7e9a0351/src/ci/run.sh#L50-L53

**Comment by Kobzol** [comment]
I think this somehow broke recently, the warnings were not there a few weeks ago, but now they're indeed spamming CI.
